src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml

```diff
@@ -13,21 +13,40 @@
 
 	<Grid Background="{StaticResource MaterialBackgroundBrush}">
 
+		<VisualStateManager.VisualStateGroups>
+			<VisualStateGroup>
+
+				<VisualState x:Name="BiggerFont">
+
+					<!-- This represents bigger phones than iPhone 5, SE -->
+					<VisualState.StateTriggers>
+						<AdaptiveTrigger MinWindowHeight="630" />
+					</VisualState.StateTriggers>
+
+					<!-- We change the font size as a workaround to respect design on small and big phones -->
```

Comment:
Workarounds in the app template?
Can't we have a design compromise? @MatFillion

Reply:
I think it is a good solution, that's how we want to do adaptative design. Let's Remove the workaround word in the comments though!